File: All_File/Coding_Practice/Armstrong.c
#include<stdio.h>
int main(){
	int rem,n=1153,i,temp,sum=0;
	temp=n;
	while(n>0){
		rem=n%10;
		sum=sum+(rem*rem*rem);
		n=n/10;
	}
	if(temp==sum){
		printf("The number armstrong number");
	}
	else
	printf("number is not armstrong number");
}

File: All_File/Deliotee_Python/Basic/Armstrong.java
public class Armstrong {
    public static void main(String[] args) {
        int n = 153;
        int temp = n, rem, sum = 0;
        while (n > 0) {
            rem = n % 10;
            sum = sum + (rem * rem * rem);
            n /= 10;
        }
        
        if (temp == sum) {
            System.out.println("this is armstron Number");
        } else {
            System.out.println("This is Not Armstrong Number");
        }
    }
}
